inc/Engine/Media/Fonts/Filesystem.php

```diff
@@ -107,6 +112,15 @@ public function write_font_css( string $font_url, string $provider ): bool {
 				}
 
 				$this->write_file( $local_path, $font_content );
+
+				$download_end  = microtime( true );
+				$download_time = $download_end - $download_start;
+
+				$download_average += $download_time;
+
+				++$count_fonts;
+
+				Logger::debug( "Font download duration -- $download_time", [ 'Host Fonts Locally' ] );
```

Comment:
I think having the font url here will make it easier for the support team to know which one is taking that much time, what do u think?